api/guides/filtering.md

```diff
@@ -4,31 +4,79 @@ title: Filtering
 
 # Filtering
 
-At times it's desirable to request a subset of records by means of a filter. Some API endpoints support the ability to filter by specific attributes. Filterable attributes will be noted on their respective index endpoint documentation areas.
+When using listing (GET) endpoints in the Adobe Experience Platform Launch API, you may find it necessary to limit the returned results to a subset of records. To accomplish this, many of the API's listing endpoints support the ability to filter by specific attributes.
 
-To filter records on an index endpoint, supply a `filter` parameter in a request
-URL, formatted as: `filter[FILTERABLE_FIELD]=value`. Filter values must be exact
-matches.
+{% alert info, Note %}
+Filters are primarily used to limit result for listing (GET) endpoints. If you wish to make structured queries to the API instead, see the guide on [searching](/api/guides/searching/).
+{% endalert %}
+
+**API format**
 
-For example:
+To filter the response for a given listing endpoint, you must supply a `filter` query parameter in the request path.
 
-`GET /companies/:company_id/properties?filter[name]=PropertyA`
+{% alert info, Note %}
+The syntax template below uses square bracket (`[]`) and space characters for readability. In practice, these characters must be URI-encoded, as outlined in [RFC 3986](https://tools.ietf.org/html/rfc3986). An example of a properly encoded request path is shown later in this guide.
 
-{% alert info, Reminder %}
-Note: The above example URI shows unencoded `[` and `]` characters simply for
-readability. In practice, these characters must be percent-encoded, per the
-requirements in RFC 3986.
+Note that if the structure of your filters are incorrect, no filters will be applied and the full result set is returned.
 {% endalert %}
 
+```http
+GET {ENDPOINT}?filter[{ATTRIBUTE_NAME}]={OPERATOR} {VALUE}
+```
+
+| Property | Description |
+| :-- | :-- |
+| `{ENDPOINT}` | A listing endpoint in the Launch API that supports filter parameters. |
+| `{ATTRIBUTE_NAME}` | The name of a specific attribute to filter results by. Keep in mind that different endpoints support different attributes for filtering. Refer to the reference guide for the endpoint you are working with for a list of available filtering attributes. |
+| `{OPERATOR}` | The operator which determines how the results are evaluated against the provided `{VALUE}`. Supported operators are listed in the [appendix](#supporter-filter-operators). |
+| `{VALUE}` | The value to compare the returned results against. When comparing for equality using the `EQ` operator, the value must be an exact, case-sensitive match in order to be included in the response. |
+
+**Request**
+
```

Comment:
I'd like to use a more useful example than filtering extension packages for pending (that'd be something probably only an extension developer would ever do).  Filtering libraries for published libraries was the first example I came across when I looked in the UI.

Also, I'd like to show the written out version before we show the Shell version.

Written out would be something like:
```http
https://reactor.adobe.io/properties/PR906238a59bbf4262bcedba248f483600/libraries?filter[state]=EQ published
```